app/models/audit/event/check.rb

```diff
@@ -14,11 +14,12 @@ def initialize(
       )
         @user = user
         @client_ip = client_ip
-        @resource = resource
```

Comment:
Is `resource` intended to be deleted? because in success scenario it seems to be called

Reply:
yes, there is a logic below that uses the same value